Makefile

```diff
@@ -1,12 +1,17 @@
 VERSION=0.0.1
+BUILD_DATE := `date +%Y-%m-%d\ %H:%M`
 SRC_FILES= main.go
 OTHER_FILES= internal/filters/filters.go \
 	     internal/artifacts/artifacts.go
 BINARY=rhc_catalog_worker
 .DEFAULT_GOAL := build
 
+LDFLAGS :=
+LDFLAGS += -X 'github.com/RedHatInsights/rhc-worker-catalog/build.Version=${VERSION}'
+LDFLAGS += -X 'github.com/RedHatInsights/rhc-worker-catalog/build.Build=${BUILD_DATE}'
+LDFLAGS += -X 'github.com/RedHatInsights/rhc-worker-catalog/build.Sha1=`git rev-parse HEAD`'
```

Comment:
@bzwei Should we just add a variable for the SHA and use it here, I don't know why I added Sha1 it should have been just Sha. Also is there a way we can get the version from the tags. Add a tag to our repo.

Reply:
@bzwei My bad it is SHA-1, so its ok.